stream.go

```diff
@@ -0,0 +1,150 @@
+package tarantool
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+type TxnIsolationLevel uint64
+
+const (
+	// By default, the isolation level of Tarantool is serializable.
+	defaultIsolationLevel = iota
+	// The ReadCommittedLevel isolation level makes visible all transactions
+	// that started commit (stream.Do(NewCommitRequest()) was called).
+	ReadCommittedLevel
+	// The ReadConfirmedLevel isolation level makes visible all transactions
+	// that finished the commit (stream.Do(NewCommitRequest()) was returned).
+	ReadConfirmedLevel
+	// If the BestEffortLevel (serializable) isolation level becomes unreachable,
+	// the transaction is marked as «conflicted» and can no longer be committed.
+	BestEffortLevel
+)
+
+type Stream struct {
+	Id   uint64
+	Conn *Connection
```

Comment:
What will happen if user close the connection and use the stream?

Reply:
In this case user gets an error `Failed to Begin: using closed connection (0x4001)` (example for `NewBeginRequest()` request call).

Reply:
I think it would be nice to add a simple test for this case.

Reply:
https://github.com/tarantool/go-tarantool/blob/5d35289ead48927fc0ff11c80aa403313dff8a00/tarantool_test.go#L2682-L2701

Reply:
Nice, thank you!